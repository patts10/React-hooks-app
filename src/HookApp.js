import React from 'react'

export const HookApp = () => {
  return (
    <>
      <h1 className='text-2xl'>HookApp</h1>
    </>
  )
}
