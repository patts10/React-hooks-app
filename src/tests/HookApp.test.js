import { HookApp } from "../HookApp";
import { shallow } from 'enzyme'


describe('Tests for <HookApp />', () => { 
  
  test('should return the component correctly', () => { 
    
    const wrapper = shallow(<HookApp />);

    expect(wrapper).toMatchSnapshot();
   });
 });