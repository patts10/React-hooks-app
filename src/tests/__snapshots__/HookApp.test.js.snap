// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for <HookApp /> should return the component correctly 1`] = `
<Fragment>
  <h1
    className="text-2xl"
  >
    HookApp
  </h1>
</Fragment>
`;
